### ISSUE TITLE
Spamming commands creates a server crash.

### ISSUE THREAD

**Issue by NiceCraftz** [LuckPerms/LuckPerms#3546]
### Description

Spamming /lpv or /lpb crashes the proxy causing the closing of the server.
https://youtu.be/bXzdksZ1g7g - Here is a link to proof

### Reproduction Steps

Just spam /lpv or /lpb using a mod (needs a lot of commands tho minimum i tested to see a bit of lag was about 1k)
I already have a precompiled mod made by me, just dm me on discord and I'll pass it to you (luckperms staff only) NiceCraftz#2657

### Expected Behaviour

Not crashing lol

### Server Details

i don't know i tested on random servers

### LuckPerms Version

see in the video

**Comment by NiceCraftz** [comment]
i tested on latest for bungeecord, it crashes so yeah it works on latest too

**Comment by LeotomasMC** [comment]
please provide a server log of this happening

**Comment by NiceCraftz** [comment]
> 
 want a 100k lines log? i already provided one of the moderators with instruction and the mod attached with src.

**Comment by lucko** [comment]
Thanks - the fix above should resolve the issue, available in `v5.4.56` and higher. (https://luckperms.net/download)

However, I should note two things:

1. Arguably, this is not a LuckPerms problem, but rather a Velocity/BungeeCord problem. At least in your configuration, the proxy is allowing players to spam command executions and not doing anything about it (rate limit, kick, etc). I tested on new-ish Paper and this is not the case, I got: `[22:44:27 INFO]: <player> lost connection: Kicked for spamming`. I incorrectly assumed that this was the case on Velocity/etc too. It might be an idea to report it to the Velocity/BungeeCord maintainers so they can consider fixing at that level too.
2. This is not a LuckPerms specific issue, and might work with other plugins/commands too. (I suspect the way LuckPerms runs commands async on another thread aplified the issue though)

Thanks again for reporting
